core/test/audits/deprecations-test.js

```diff
@@ -44,6 +44,7 @@ describe('Deprecations audit', () => {
             },
           },
           {
+            // Does not exist anymore.
```

Comment:
Why keep this around, doe we still need to handle it gracefully in older Chrome versions?

Reply:
Including a type that used to exist but no longer does is a minor but possibly useful test case. I didn't see a reason to remove it, but you're right it is not immensely valuable.